nylas/models/messages.py

```diff
@@ -70,30 +53,28 @@ class Message:
         attachments: The attachments on the message.
         folders: The folders that the message is in.
         headers: The headers of the message.
+        created_at: Unix timestamp of when the message was created.
     """
 
     id: str
     grant_id: str
-    thread_id: str
-    subject: str
-
     from_: List[EmailName] = field(metadata=config(field_name="from"))
-
     date: datetime
-
-    unread: bool
-    starred: bool
-
-    snippet: str
-    body: str
-
+    object: str = "messages"
```

Comment:
should this be `message` or is it actually plural? :thinking:

Reply:
Great catch, will fix. Thanks!